src/examples/bus-service-geometry/main.js

```diff
@@ -0,0 +1,56 @@
+import $ from 'jquery'
+import L from 'leaflet'
+
+// To make live requests get your app_id and app_key by signing up at https://developer.transportapi.com/signup
+// and filling them here
+const appId = ''
+const appKey = ''
+
+// Specify a bounding box
+const minLon = -3.430
+const minLat = 50.610
+const maxLon = -3.370
+const maxLat = 50.650
+
+const url = (appId === '' || appKey === '')
+  ? 'response.json'
+  : 'https://transportapi.com/v3/uk/bus/services.json?edge_geometry=true' +
+    `&app_id=${appId}&app_key=${appKey}` +
+    `&min_lon=${minLon}&min_lat=${minLat}&max_lon=${maxLon}&max_lat=${maxLat}`
+
+$.getJSON(url, data => {
+  const map = drawMap()
+  const services = data.member
+  drawGeometry(services, map)
+})
+
+function drawMap () {
+  const map = L.map('mapElement').setView([51.505, -0.09], 13)
+  const urlTemplate = 'https://{s}.basemaps.cartocdn.com/light_all/{z}/{x}/{y}{r}.png'
+  map.addLayer(L.tileLayer(urlTemplate, {
+    attribution: '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors &copy; ' +
+        '<a href="https://carto.com/attributions">CARTO</a>',
+    subdomains: ['a', 'b', 'c', 'd'],
+    maxZoom: 19,
+    id: 'map',
+    tileSize: 512,
+    zoomOffset: -1
+  }))
+  return map
+}
+
+function drawGeometry (services, map) {
+  const edgesLayerGroup = L.layerGroup()
+  map.addLayer(edgesLayerGroup)
+  // The coordinates are returned as [longitude, latitude] so they need to be swapped before they're passed to Leaflet
+  const polyLines = services
```

Comment:
Because if I draw a polyline out of all edges there will be a straight line between the end of the geometry of service i and the start of the geometry of service i+1 and this is not correct.

Reply:
You have multiple services, each has multiple edges, each edge has multiple lines. Right now each Leaflet `Polyline` represents a single edge. What I'm suggesting is that it represent a single service, not all services

Reply:
Unfortunately, it seems that for some services, the geometry is not a continuous polyline. This is an example of one such service: https://kanbanzone-attachment-prod.s3.amazonaws.com/uploads/5d9d9b6f687b9600154451fc/5dc4114c9685f900156cd0fd/62effd7a2a17c30017694432/k6IDL921r78OA2bT/service-geometry.png
This is a little bit contra intuitive to me, I will ask Dave about it.